lblod/frontend-gelinkt-notuleren
# Update template UUID instantiator

### Overview
This PR updates the following packages:
- editor-plugins to https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/507 (dev-commit)
- `@lblod/template-uuid-instantiator` to https://github.com/lblod/template-uuid-instantiator/pull/2 (dev-commit)

Updating these packages fixes an issue where template instantiation did not happen correctly. Imported resources should now work as expected with this PR.
x

##### connected issues and PRs:
https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/507
https://github.com/lblod/template-uuid-instantiator/pull/2

### Setup
Ensure the frontend is connected with the correct RB env (which contains templates with imported resources)

### How to test/reproduce
- Start the frontend
- Login in as any user
- Create a new agendapoint based on a template containing a snippet placeholder with imported resources
- Ensure the `data-imported-resources` attribute of the snippet placeholder is correct
- Inserting a snippet should work (and the imported resources should automatically connect)

### Checks PR readiness
